### DISCUSSION TITLE
How to send webhook after comment?

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#9266] [Archive]
Hello! I want to send webhook, every time user makes comments to the item of specific collection.

Is it possible?

**Comment by azrikahar** [comment]
Every time a user makes a comment, it's added to the Activity system table. Since [Directus do provide hook for `activity`](https://docs.directus.io/extensions/hooks/#event-format-options), so yes it is possible 👍

Here's a short and simple custom hook. In my case, I added it at `extensions/hooks/comments-hook/index.js`:

```js
module.exports = function registerHook() {
	return {
		'activity.create': function ({ payload }) {
			if (payload.action !== 'comment') return;

			console.log(`New comment added: `, payload);
		},
	};
};
```

the if statement used above is just to prevent any other activity that are NOT comments from doing anything (unless you want them to).

And here's a short clip to show it in action:

https://user-images.githubusercontent.com/42867097/139569729-6288de31-877c-441c-b286-ea8b17974691.mp4

**Reply by IlyaPetrovM** [reply]
Thank you! 

I'll try your extension, but I do not quite understand, is it possible to set up this hook make POST query to **custom** url? Where should I write this url?

**Reply by azrikahar** [reply]
You can check out example shown in the custom API hooks documentation: https://docs.directus.io/extensions/hooks/#full-example

The following line in that example code should be what you're looking for (inside the try block):

```js
    await axios.post('https://example.com/recipes', input);
```

**Reply by IlyaPetrovM** [reply]
Thank you!
